src/MassTransit/Configuration/Registration/TransportBusInstance.cs

```diff
@@ -18,5 +26,38 @@ public TransportBusInstance(IBusControl busControl, IHostConfiguration hostConfi
         public IBusControl BusControl { get; }
 
         public IHostConfiguration HostConfiguration { get; }
+
+        public void Connect(IBusAttachment attachment)
+        {
+            _attachments.Add(attachment);
+        }
+
+        public async Task Start(CancellationToken cancellationToken)
+        {
+            LogContext.SetCurrentIfNull(HostConfiguration.LogContext);
+
+            LogContext.Info?.Log("Starting bus: {Type} instance", InstanceType.Name);
+            await BusControl.StartAsync(cancellationToken).ConfigureAwait(false);
+
+            if (!_attachments.Any())
+                return;
+
+            LogContext.Info?.Log("Connecting bus ({Type}) attachments: {Names}", InstanceType.Name, string.Join(", ", _attachments.Select(x => x.Name)));
```

Comment:
Same comment as multi-bus. Isn't multi-bus a delegated bus instance? Seems we could dry this up and have it in one place.